### ISSUE TITLE
If it is possible to add the functionality to disable notifications for certain chats (lobby)

### ISSUE THREAD

**Issue by Skyp93** [RetroShare/RetroShare#2571]
If it is possible to add the functionality to disable notifications for certain chats (lobby). 
Without saving the settings. Only in runtime?

**Comment by defnax** [comment]
yes i wanted some functionality too to disable notify, its possible on social apps/platforms.
Bad at the moment you cant disable the notify.

on whatsapp/Telegram you can set to mute then you doesnt get more notifyd for new chats from some persons/chatgroups
